libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -977,7 +983,7 @@ protected Void doInBackground(Void... nothings) {
 	        try {
 	        	OAuth2.revokeRefreshToken(HttpAccess.DEFAULT, new URI(loginServer), refreshToken);
 	        } catch (Exception e) {
-	        	Log.w("SalesforceSDKManager:revokeToken", e);
```

Comment:
I guess we should do that across the board - the call stack will show the method where the exception took place - so there is no need to have the method name in the tag